src/Mirco-MMerce/Services/Catalog/Catalog.API/appsettings.Development.json

```diff
@@ -0,0 +1,13 @@
+{
+  "DatabaseSettings": {
+    "ConnectionString": "mongodb://localhost:27017",
```

Comment:
ConnectionString was different in VS Docker debug pointer: "mongodb://catalogdb:27017"
Where catalogdb is docker container name of the mongo database